src/main/java/net/william278/velocitab/config/Placeholder.java

```diff
@@ -107,11 +110,20 @@ private static String getPlaceholderFallback(@NotNull Velocitab plugin, @NotNull
         return "";
     }
 
-    public static CompletableFuture<String> replace(@NotNull String format, @NotNull Velocitab plugin,
-                                                    @NotNull TabPlayer player) {
+    @NotNull
+    public static String replaceInternal(@NotNull String format, @NotNull Velocitab plugin,
+                                         @Nullable TabPlayer player) {
+
+        boolean foundRelational = false;
+        if(format.contains("<vel")) {
```

Comment:
I do wonder if there's some better logic we could be using to represent this stuff rather than embedded string bits. perhaps making our placeholder a string wrapper with some booleans (index to start/end of player name in string)? Not sure.

Reply:
We'll do that for another release